ludwig/backend/ray.py

```diff
@@ -822,7 +834,9 @@ def provision_preprocessing_workers(self):
             yield
         else:
             num_cpu = self._preprocessor_kwargs["num_cpu_workers"]
-            self._preprocessor_pg = placement_group([{"CPU": num_cpu}])
+            bundles = self.generate_bundles(num_cpu)
```

Comment:
This comment is for the code above, but we shouldn't be logging a warning if the user doesn't set this flag (warnings indicate there might be something wrong, but here it's just that some option wasn't enabled).

I would also remove the `use_preprocessing)placement_group` param, since it looks like this should be assumed when `num_cpu_workers` is set. So we can just check for the existence of `num_cpu_workers` instead.

Finally, I would consider renaming this param to just `num_cpus` since we're just asking for a pool of CPUs, not specifically a number of actors, etc.

Reply:
done